frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss

```diff
@@ -17,10 +17,18 @@
 
     &:hover:not([aria-disabled='true']) {
         border-color: var(--primary-3000-hover);
+
+        .posthog-3000 & {
+            border-color: var(--border-bold);
+        }
     }
 
     &.LemonInput--focused:not([aria-disabled='true']) {
         border-color: var(--primary-3000);
+
+        .posthog-3000 & {
+            border-color: var(--border-bold);
```

Comment:
I assume this changes everywhere in 3000? Feels like a big change but understand the concerns about it looking like an error state. Just wondering if grey is the greatest active state

Reply:
I think it is miles better than orange (i.e. nearly red). So yeah I would vote for it as the default until we have a better option